pages/gasStations.js

```diff
@@ -0,0 +1,69 @@
+import Layout from '../components/layout'
+import styles from "../styles/style.module.css"
+import React, { useEffect, useState } from 'react';
+import Link from "next/link"
+import { useRouter } from 'next/router';
+function fecha(date){
+    return new Date(date).toLocaleString();
+  }
+
+
+export default function PageRestaurants () {
+  const [items, setItems] = useState([]);
+  
+    useEffect(() => {
+      fetch('http://turismo:8081/rest/gasStations')
+        .then(response => response.json())
+        .then(json => setItems(json))
+          .catch(error=>alert("Error"+error.message))
+    }, [])
+  
+  return (
+    <Layout>
+      <div className={styles.cajaCentral}>
+        <div className={styles.cajaSuperior}>
+          <h1>Gas Stations</h1>
+        </div>
+        <div className={styles.cajaTriple}>
+            {items.map((item, index) => {
+                console.log(item);
+                if (item['label'] == "REPSOL"){
```

Comment:
This if else is a SMELL CODE. so I'm gonna give you a kind of clue to solve this.


1.- Create an object with the urls
2. Get the value of the object with the disered key.

Exmaple

constants.js file ->

const IMAGES_GASSTATION = {
 GALP: https://www.telefonodirecto.es/wp-content/uploads/2017/07/Tel%C3%A9fono-de-Galp-Espa%C3%B1a.png,
CARREFOUR: https://www.bitrefill.com/content/cn/b_rgb%3AFFFFFF%2Cc_pad%2Ch_600%2Cw_1200/v1578400400/carrefour-france.jpg
}


in the component


import {IMAGES_GASSTATION} from "constants";

items.map((item, index) => {
...
const image = IMAGES_GASSTAION[item['label']];
...
}

Reply:
I created an constant document where put image url of different gas stations. After i put a comparative into document